dashboard/app/controllers/api_controller.rb

```diff
@@ -350,7 +350,7 @@ def load_section
   end
 
   def load_script
-    @script = Script.get_from_cache(params[:script_id]) if params[:script_id].present?
-    @script ||= @section.script || Script.twenty_hour_script
+    script_id = params[:script_id] || @section.script.id
```

Comment:
Is it possible for `params[:script_id]` to be the empty string? If so, I need to change this line to skip over it.